cctrl/user_commands.py

```diff
@@ -72,6 +72,30 @@ def parse_cmdline(user):
         help="the password")
     create_subparser.set_defaults(func=user.create)
 
+    setup_subparser = subparsers.add_parser('setup', help="setup user")
+    setup_subparser.add_argument(
+        "--email",
+        nargs='?',
```

Comment:
the flag itself is optional; but if given the value should not be optional (https://docs.python.org/3/library/argparse.html#nargs)